src/components/select/types.ts

```diff
@@ -0,0 +1,18 @@
+export enum SelectSize {
+    Normal = '',
+    Small = 'select-sm',
+}
+
```

Comment:
co je to enum prosím tě?

Reply:
Wiki definici z hlavy nedam ale je to specialni typ objektu, kterej ti definuje set nejakych spolecnych hodnot. Nad enumem se da iterovat, takze se to chova trochu jako pole a zaroven muzes pristupovat k jednotlivym hodnotam jako v objektu. V typescriptu ti to indikuje jakych hodnot muze dana promenna, parametr atd nabyvat hodnot. Kdyz pridas neco jinyho, bude to rvat.